index.ts

```diff
@@ -205,6 +208,22 @@ module.exports = {};`
     return Object.keys(this._devDependencies).map(dependency => this._devDependencies[dependency]);
   }
 
+  linkDependency(name: string, link?: string) {
+    const root = path.join(this.root, this.name, 'node_modules')
+    const basedir = path.dirname(getCallerFile());
```

Comment:
This strikes me a sortof odd actually. If a relative path to the link target is passed, I would expect it to be from `process.cwd()`?

Reply:
The goal for this was actually to support not relative, but node_module's by name.

For relative paths you are right, we should special case it to be cwd.